src/test/java/uk/ac/cam/cl/dtg/isaac/api/IsaacIntegrationTest.java

```diff
@@ -317,16 +317,17 @@ public static void setUpClass() {
         new EmailService(emailManager, groupManager, userAccountManager), gameManager, properties);
 
     quizManager = new QuizManager(properties, new ContentService(contentManager), contentManager,
-        new ContentSummarizerService(mapperFacade, new URIManager(properties)), contentMapper);
+        new ContentSummarizerService(mapperFacade, new URIManager(properties)), contentMapperUtils);
     quizAssignmentPersistenceManager = new PgQuizAssignmentPersistenceManager(postgresSqlDb, mapperFacade);
     quizAssignmentManager = new QuizAssignmentManager(quizAssignmentPersistenceManager,
         new EmailService(emailManager, groupManager, userAccountManager), quizManager, groupManager, properties);
     assignmentService = new AssignmentService(userAccountManager);
     quizAttemptPersistenceManager = new PgQuizAttemptPersistenceManager(postgresSqlDb, mapperFacade);
     quizAttemptManager = new QuizAttemptManager(quizAttemptPersistenceManager);
-    quizQuestionAttemptPersistenceManager = new PgQuizQuestionAttemptPersistenceManager(postgresSqlDb, contentMapper);
+    quizQuestionAttemptPersistenceManager = new PgQuizQuestionAttemptPersistenceManager(postgresSqlDb,
+        contentMapperUtils);
     quizQuestionManager =
-        new QuizQuestionManager(questionManager, contentMapper, quizQuestionAttemptPersistenceManager, quizManager,
+        new QuizQuestionManager(questionManager, mapperFacade, quizQuestionAttemptPersistenceManager, quizManager,
```

Comment:
Doesn't this mean that the `QuizQuestionManager` should be accepting a `ContentMapperUtils` instead of the main mapper?

Reply:
The `QuizQuestionManager` was previously using the contentMapper for the `getAutoMapper()` method. That method no longer exists and the manager just takes the mapper instance itself.